docs/contracts.rst

```diff
@@ -301,10 +301,10 @@ inheritable properties of contracts and may be overridden by derived contracts.
 
 ::
 
-    pragma solidity ^0.4.22;
+    pragma solidity ^0.4.24;
```

Comment:
This still uses ``^`` - but actually the ``constructor`` syntax was already introduced a while ago, so ``^`` should be fine, shouldn't it :-)?

Reply:
I'd also change this to ``>0.4.24`` so that we do not necessarily need to touch this before the release (using ``^0.4.24`` would lead to a compilation error once 0.5.0 is released, wouldn't it?)

Reply:
Makes sense.